include/ignition/fuel_tools/FuelClient.hh

```diff
@@ -177,7 +177,7 @@ namespace ignition
       /// \brief Remove a model from ignition fuel
       /// \param[in] _id The model identifier.
       /// \return Result of the delete operation
-      /// Deprecate this function in ign-fuel-tools5. DeleteUrl
+      /// Deprecate this function in ign-fuel-tools6. DeleteUrl
```

Comment:
Should this function already be deprecated?

Reply:
It sounds like we never got to it on v5. I'll open a PR targeting v6 now.

Reply:
See https://github.com/ignitionrobotics/ign-fuel-tools/pull/159